package.json

```diff
@@ -23,58 +23,56 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "ansi-colors": "^3.2.4",
     "brfs": "^2.0.2",
     "browserify": "^16.2.3",
     "chai": "^4.2.0",
     "connect": "3.6.6",
-    "del": "^4.1.0",
-    "envify": "4.1.0",
+    "del": "^4.1.1",
     "eslint": "^5.16.0",
     "eslint-config-braintree": "2.0.0",
-    "finalhandler": "1.1.1",
-    "gulp": "3.9.1",
-    "gulp-autoprefixer": "^6.0.0",
-    "gulp-brfs": "0.1.0",
-    "gulp-clean-css": "^4.0.0",
-    "gulp-envify": "1.0.0",
+    "fancy-log": "^1.3.3",
+    "gulp": "^4.0.2",
+    "gulp-autoprefixer": "^6.1.0",
+    "gulp-clean-css": "^4.2.0",
     "gulp-less": "^4.0.1",
     "gulp-rename": "^1.4.0",
     "gulp-replace": "^1.0.0",
     "gulp-size": "3.0.0",
     "gulp-streamify": "1.0.2",
     "gulp-uglify": "^3.0.2",
-    "gulp-util": "3.0.8",
     "is-uuid": "1.0.2",
     "jsdoc": "^3.6.1",
     "jsdoc-template": "braintree/jsdoc-template#3.2.0",
-    "karma": "^4.0.1",
+    "karma": "^4.1.0",
     "karma-browserify": "^6.0.0",
     "karma-mocha": "1.3.0",
     "karma-mocha-reporter": "2.2.5",
     "karma-phantomjs-launcher": "1.0.4",
     "mkdirp": "0.5.1",
     "mocha": "^6.1.4",
     "phantomjs-prebuilt": "2.1.16",
-    "run-sequence": "2.2.1",
-    "sauce-connect-launcher": "0.14.0",
+    "sauce-connect-launcher": "^0.17.0",
     "serve-static": "1.13.2",
-    "sinon": "^7.3.1",
+    "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0",
-    "vinyl-source-stream": "2.0.0"
+    "through2": "^3.0.1",
+    "vinyl": "^2.2.0",
+    "vinyl-source-stream": "^2.0.0",
+    "watchify": "^3.11.1"
   },
   "dependencies": {
-    "braintree-web": "3.44.2",
     "@braintree/asset-loader": "0.2.1",
     "@braintree/browser-detection": "1.7.0",
     "@braintree/class-list": "0.1.0",
-    "promise-polyfill": "8.1.0",
-    "@braintree/wrap-promise": "1.1.1"
+    "@braintree/wrap-promise": "^2.0.0",
```

Comment:
This shouldn't be updated until the underlying braintree-web version is updated with this same version. See https://github.com/braintree/braintree-web-drop-in/pull/495

Reply:
i think this was actually an accident—i think my brain autocompleted it to the eslint thing (which btw i will be version-bumping and giving a peer dev of eslint@5)